src/admin/utils/fixMultipleSourceAssigningAuthority.js

```diff
@@ -63,13 +63,37 @@ function fixPractitionerResource(resource, fixMultipleOwners) {
     return resource;
 }
 
+function fixResource(resource) {
+    let security = resource.meta.security || [];
```

Comment:
Ideally, security tag should be available but if not present, it should skip the whole process and add logger for the doc ID.